package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-openshift-templates",
   "description": "OpenShift V3 Templates for FH MBaaS",
-  "version": "0.24.87",
+  "version": "0.24.88",
```

Comment:
@maleck13 I agree with you that it probably doesn't matter. OTOH, this is an open source project now, so we don't know how it might be used. :) If we're going to use semver-looking versions, we may as well stick to semver. Otherwise, why not just have a single incrementing number?